akka-cluster/src/multi-jvm/scala/akka/cluster/ClusterDeathWatchSpec.scala

```diff
@@ -138,23 +141,6 @@ abstract class ClusterDeathWatchSpec
 
     }
 
-    "receive Terminated when watched path doesn't exist" ignore {
```

Comment:
Did you mean to remove this?

Reply:
I did, it has been ignored for 4 years so thought it may as well be deleted rather than unwarned

Reply:
oh yeah :laughing: